src/main/kotlin/com/toasttab/pulseman/pulsar/filters/AuthClassFilter.kt

```diff
@@ -28,10 +29,10 @@ import java.net.URLClassLoader
 class AuthClassFilter : ClassFilter<PulsarAuthHandler> {
     override fun getClasses(file: File): Set<PulsarAuthHandler> {
         val classLoader = URLClassLoader(arrayOf(file.toURI().toURL()))
-        val resultSet = mutableSetOf<PulsarAuthHandler>()
-        Reflections(classLoader).getSubTypesOf(Authentication::class.java).forEach {
-            resultSet.add(PulsarAuthHandler(it, file))
-        }
-        return resultSet
+        return Reflections(classLoader.urLs)
```

Comment:
Is the `urLs` a typo?

Reply:
Howd I not notice that, its from the java to kotlin conversion for getter functions, this call is to a third party lib
So it's calling

```java
    public URL[] getURLs() {
        return ucp.getURLs();
    }
```
I'm not to sure on the auto conversion logic, but it doesn't seem to like a mix or acronyms and plurals